### PR TITLE
NOTIF-190 Fix Dockerfile

### PR DESCRIPTION
**Pull request by theute**
This builds on my machine

### CODE REVIEW

**Comment by bennetelli** [comment]
Mom, want to take a look what causes my issue.

File: docker/Dockerfile-build.jvm [hunk 1]
```diff
@@ -5,13 +5,9 @@
...
 USER root
-COPY pom.xml /home/jboss
-COPY backend/src /home/jboss/backend/src
-COPY backend/pom.xml /home/jboss/backend
-COPY checkstyle/checkstyle/checkstyle.xml /home/jboss/backend
-COPY checkstyle/checkstyle/checkstyle-suppressions.xml /home/jboss/backend
+COPY . /home/jboss
 WORKDIR /home/jboss
-RUN mvn clean package -DskipTests --projects :notifications-parent,:notifications-backend
+RUN mvn clean install -DskipTests
```
Comment (bennetelli) [inline docker/Dockerfile-build.jvm:10]: I think install isn't needed here. I would say we don't need the dependencies to be in the local .m2/repository folder and package is enough to work with it.

my issue was that I added checktyle at the --projects flag, but in the wrong order.

**Review by bennetelli — COMMENTED** [review]
RUN mvn clean package -DskipTests -Dskip.yarn --projects :checkstyle,:notifications-parent,:notifications-backend

This works as well and should take less time and memory I would say.

**Review by bennetelli — APPROVED** [review]
You can take yours or mine. Whatever you like :)

**Comment by theute** [comment]
Go for it, I'll close this one.
As long as you don't duplicate checkstyles ;-)

**Comment by theute** [comment]
But don't skip yarn, we need it

**Pull request closed without merging**